librosa/core/constantq.py

```diff
@@ -62,9 +63,9 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
```

Comment:
I'm not sure how readability could be improved for that particular op; it's already commented and correctly does what it claims.  I'm inclined to leave it as is.